src/Components/creatorPage/IllustratedWorkCard.js

```diff
@@ -4,8 +4,8 @@ import propTypes from 'prop-types';
 
```

Comment:
Why do the authored and illustrated work cards exist? Can you consolidate them or replace with the book card entirely?